### PR TITLE
fix to return null instead of undefined

### PR DESCRIPTION
**Pull request by bomsy**
Associated Issue: #5023

### Summary of Changes

* return `null` from instead of `undefined` which broke react rendering

### Test Plan

Tell us a little a bit about how you tested your patch.

Example test plan:

Steps to Reproduce:

- go to https://jovandemo.modulentwicklung.de:8001/admin
- login with User: anyone@mozilla.org PW: 123456789
- go to Orders > Orders
- Open Debugger and search for file: Shared/Directives/DataTable/DataTableController.ts
- Set a breakpoint in line 197
 - Move cursor inside the view-port to trigger angular

### Screenshots/Videos (OPTIONAL)

image

### CODE REVIEW

**Comment by jasonLaster** [comment]
how was it breaking?

File: assets/images/Svg.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ const svg = {
   nextjs: require("./nextjs.svg"),
   showSources: require("./showSources.svg"),
   showOutline: require("./showOutline.svg"),
-  nuxtjs: require("./nuxtjs.svg")
+  nuxtjs: require("./nuxtjs.svg"),
+  rxjs: require("./rxjs.svg")
```
Comment (codehag) [inline assets/images/Svg.js]: interesting, why are we adding this back in?

Comment (bomsy) [reply assets/images/Svg.js]: We have an issue which was looking to render the `rxjs` svg and couldn't find it so it return `undefined` which broke rendering. The svg file but it was not included in the map above.

Comment (bomsy) [reply assets/images/Svg.js]: Not sure why the tests are failing ..

Comment (jasonLaster) [reply assets/images/Svg.js]: no worries. i've got it

**Comment by TitanNano** [comment]
@jasonLaster how long until this will land in nightly?

**Comment by jasonLaster** [comment]
it will land tomorrow.

**Pull request merged**